### ISSUE TITLE
After successfully cancelling Axios, will the 'then' callback still execute?

### ISSUE THREAD

**Issue by liangliangGit** [axios/axios#6272]
### Describe the issue

The following is my code. After successfully canceling Axios, will the log (test then) in 'then' still be printed to the console? In other words, will 'then' still execute?
`AXIOS.post(`test`, data,{
  cancelToken: cancelSource.cancelToken,
}).then(response => {

  console.log("test then")
}).catch(error => {

})`

**Comment by startjava** [comment]
https://axios-http.com/zh/docs/cancellation

now use signal: controller.signal.

cancelToken is old !

**Comment by liangliangGit** [comment]
> https://axios-http.com/zh/docs/cancellation
> 
> now use signal: controller.signal.
> 
> cancelToken is old !

Thank you for your response. I will proceed with the latest approach, but I am still curious about the answer to my previous question: will the 'then' be executed? @startjava

**Comment by liangliangGit** [comment]
Successfully cancelled Axios, but why did the 'then' continue to execute? Here is the code:

`const controller = new AbortController();

axios.get('/foo/bar', {
   signal: controller.signal
}).then(function(response) {
   //...
});

controller.abort()`

**Comment by liangliangGit** [comment]
> https://axios-http.com/zh/docs/cancellation
> 
> now use signal: controller.signal.
> 
> cancelToken is old !

Successfully cancelled Axios, but why did the 'then' continue to execute? Here is the code:
const controller = new AbortController();

axios.get('/foo/bar', {
   signal: controller.signal
}).then(function(response) {
   //...
});
controller.abort()

**Comment by startjava** [comment]
check response param about propperties

**Comment by DigitalBrainJS** [comment]
No, it won't. Cancellation in Axios, as in fetch, means promise rejection with a special error instance, so only the callback of the catch chain will be invoked.
